src/aal/aal_arm.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#if defined(__arch64__)
+#  define SNMALLOC_VA_BITS_64
+#else
+#  define SNMALLOC_VA_BITS_32
+#endif
+
+#include <iostream>
+namespace snmalloc
+{
+  /**
+   * ARM-specific architecture abstraction layer.
+   */
+  class AAL_arm
+  {
+  public:
+    /**
+     * Bitmap of AalFeature flags
+     */
+    static constexpr uint64_t aal_features = IntegerPointers | NoCpuCycleCounters;
```

Comment:
Another Clangformat issue.  I don't know why it didn't report this one previously.

```suggestion
    static constexpr uint64_t aal_features =
      IntegerPointers | NoCpuCycleCounters;
```